File: Chorme_extension/contentScript/All-in-JPEG/content-info/TextInfo.js
export default class TextInfo {
    constructor(text) {
        this.offset = 0;
        this.dataSize = 0;
        this.data = '';
        this.attribute = 0;

        this.init(text);
    }

    init(text) {
        this.dataSize = text.data.length * 2;
        this.data = text.data;
        this.attribute = text.contentAttribute.code;
    }

    getTextInfoSize() {
        // Int(4) * 3
        return 4 * 3;
    }
}

File: OnePIC/app/src/main/java/com/goldenratio/onepic/AllinJPEGModule/ContentInfo/TextInfo.kt
package com.goldenratio.camerax.PictureModule.Info

import com.goldenratio.onepic.AllinJPEGModule.Content.Text

class TextInfo(text: Text) {
    var offset : Int = 0
    var dataSize : Int = 0
    var data : String
    var attribute : Int

    init {
        dataSize = (text.data.length)*2
        data = text.data
        attribute = text.contentAttribute.code
    }

    fun getTextInfoSize() : Int{
        // Int(4) X 3
        return 4*3
    }
}